internal/gatewayapi/route.go

```diff
@@ -173,6 +173,15 @@ func (t *Translator) processHTTPRouteRules(httpRoute *HTTPRouteContext, parentRe
 		dstAddrTypeMap := make(map[ir.DestinationAddressType]int)
 
 		for _, backendRef := range rule.BackendRefs {
+			if len(backendRef.Filters) > 0 {
+				parentRef.SetCondition(httpRoute,
+					gwapiv1.RouteConditionResolvedRefs,
+					metav1.ConditionFalse,
+					"UnsupportedRefValue",
+					"The filters field within BackendRef is not supported",
+				)
+			}
```

Comment:
probably also need to add this for tcproute, tlsroute and udproute ?

Reply:
thought so, but they don't have it https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.BackendRef

Reply:
thanks for checking !